e2e/pages/app/input-dropdown/input-dropdown.testpage.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="row">
+    <div class="col-md-5">
+        <form [formGroup]="form" novalidate>
+            <div class="form-group m-nil">
+
+        <ux-input-dropdown
+            [(selected)]="selected"
+            id="dropdown"
+            [dropdownOpen]="dropdownOpen"
+            [disabled]="disabled"
+            (dropdownOpenChange)="dropdownOpenChange($event)"
+            [(filter)]="filter"
+            [allowNull]="allowNull"
+            [maxHeight]="maxHeight"
+            [placeholder]="placeholder"
+            aria-label="Filter input"
+            formControlName="inputForm">
+
+            <ng-template #displayContent>
```

Comment:
This template is not used.

Reply:
The template is being used, if it is removed then the input dropdown doesn't display properly
![disabledbtn](https://user-images.githubusercontent.com/56357626/104353951-67840b80-5500-11eb-939b-35d02dd95404.PNG)

Reply:
Yes you are right, I misread this.